packages/Textarea/src/textarea.scss

```diff
@@ -2,94 +2,156 @@
-    .ids-textarea--focus & {
-      border-color: var(--ids-textarea-focus-border-color);
-    }
+.ids-textarea {
+    font-family: var(--ids-textarea-font-family);
+    font-size: var(--ids-textarea-font-size);
+    position: relative;
 
-    &--error {
-      border-color: var(--ids-textarea-error-border-color);
+    &,
+    & * {
+        box-sizing: border-box;
     }
 
-    &:disabled {
-      background: var(--ids-textarea-disabled-background);
-      border-color: var(--ids-textarea-disabled-background);
-      color: var(--ids-textarea-disabled-color);
-      cursor: not-allowed;
+    &__field {
+        border: .1rem solid var(--ids-textarea-border-color);
+        border-radius: var(--ids-textarea-border-radius);
+        color: var(--ids-textarea-color);
+        display: block;
+        font-family: inherit;
+        font-size: inherit;
+        line-height: var(--ids-textarea-line-height);
+        outline: none;
+        padding: var(--ids-textarea-padding);
+        resize: none;
+        width: 100%;
+
+        @include mixins.fancy-webkit-scrollbar;
+
+        .ids-textarea--has-char-count & {
+            padding-bottom: var(--ids-textarea-char-indicator-padding-bottom);
+            scroll-padding-bottom: var(--ids-textarea-char-indicator-padding-bottom);
+        }
+
+        &::placeholder {
+            color: var(--ids-textarea-color-placeholder);
+        }
+
+        &:focus,
+        .ids-textarea--focus & {
+            border-color: var(--ids-textarea-focus-border-color);
+            box-shadow: var(--ids-textarea-focus-box-shadow);
+        }
+
+        &:hover:focus,
+        .ids-textarea--focus:hover & {
+            border-color: var(--ids-textarea-focus-border-color);
+        }
+
+        &:hover {
+            background-color: var(--ids-textarea-hover-background-color);
+            border-color: var(--ids-textarea-hover-border-color);
+        }
+
+        &--error {
+            border-color: var(--ids-textarea-error-border-color);
+        }
+
+        &--error:hover,
+        &--error:focus,
+        &--error:focus:hover {
+            border-color: var(--ids-textarea-error-border-color);
+        }
+
+        &:disabled {
+            background: var(--ids-textarea-disabled-background);
+            border-color: var(--ids-textarea-disabled-background);
+            color: var(--ids-textarea-disabled-color);
+            cursor: not-allowed;
+        }
     }
 
-    &--has-char-count {
-      padding-bottom: var(--ids-textarea-char-indicator-padding-bottom);
+    &__char-indicator {
+        bottom: var(--ids-textarea-char-indicator-bottom);
+        font-size: var(--ids-textarea-char-indicator-size);
+        line-height: var(--ids-textarea-char-indicator-line-height);
+        position: absolute;
+        right: var(--ids-textarea-char-indicator-right);
     }
-  }
-
-  &__char-indicator {
-    bottom: var(--ids-textarea-char-indicator-bottom);
-    color: var(--ids-textarea-char-indicator-color);
```

Comment:
Do we not need a color anymore? If this is not needed, please remove the variable at the top.

Reply:
good catch I removed it by mistake.